change/@fluentui-react-switch-504a98f0-bdbd-4377-959c-1dcb2275040a.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: use `useFocusWithin` hook for :focus-within styles",
```

```suggestion
  "comment": "chore: use `useFocusWithin` hook for :focus-within styles",
```

Feel free to ignore, but IMO this and all other change files need to be updated to use `chore` as where is a fix? 🐱 It worked before, right?

---

I thought that `chore` didn't really reflect that this is quite a big change. Would `feat` be more appropriate ?

---

I would go with [`refactor` then](https://www.conventionalcommits.org/en/v1.0.0/#summary)